app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/parser/AccessibilityParserImpl.kt

```diff
@@ -29,10 +31,19 @@ class AccessibilityParserImpl : AccessibilityParser {
                 rootNode
                     .findAccessibilityNodeInfosByViewId("$packageName:id/$viewId")
                     .map { accessibilityNodeInfo ->
-                        browser.urlExtractor(accessibilityNodeInfo.text.toString())
+                        browser
+                            .urlExtractor(accessibilityNodeInfo.text.toString())
+                            ?.trim()
+                            ?.let { rawUrl ->
+                                if (rawUrl.contains(other = ".") && !rawUrl.hasHttpProtocol()) {
```

Comment:
the `.` check is just to make sure that the URL has a valid domain following pattern of `foo.bar/whatever` ?

Reply:
That is my assumption, but honestly, I am not 100% sure. This logic was based on the code [here](https://github.com/bitwarden/mobile/blob/main/src/App/Platforms/Android/Accessibility/AccessibilityHelpers.cs#L425)